File: KH.Pepper.Services/DTO/ProductPriceDto.cs
using AutoMapper;
using KH.Pepper.Core.Domain;

namespace KH.Pepper.Core.AppServices.Dto
{

    [AutoMap(typeof(ProductPrice), ReverseMap = true)]
    public class ProductPriceDto
    {

        public int Id { get; set; }
        public int ProductId { get; set; }

        
        public int ProductQtyId { get; set; }

        
        public decimal? Price { get; set; }

        public Boolean? IsOfferAvailable { get; set; }

        public decimal? OfferAmount { get; set; }

        public String OfferDetails { get; set; }

        public Boolean? IsDisplayOnProduct { get; set; }
 

    }
}

File: KH.Pepper.Web.UI/src/app/services/api/models/productpriceDto.ts

export class ProductPriceDto { 
    Id : number = 0;
    ProductId : number = 0;
    ProductQtyId : number = 0;
    Price:number = 0.00;
    IsOfferAvailable:boolean = false;
    OfferAmount:number = 0.00;
    OfferDetails:string = '';
    IsDisplayOnProduct:boolean = false;
   
}